shelley/chain-and-ledger/executable-spec/src/STS/Epoch.hs

```diff
@@ -39,16 +43,33 @@ initialEpoch :: InitialRule (EPOCH hashAlgo dsignAlgo)
 initialEpoch =
   pure $ EpochState emptyAccount emptySnapShots emptyLedgerState emptyPParams
 
+votedValuePParams
+  :: PPUpdate dsignAlgo
+  -> PParams
+  -> Maybe PParams
+votedValuePParams (PPUpdate ppup) pps =
+  let
+    votes = Map.foldr
+              (\vote tally -> Map.insert vote (Map.findWithDefault 0 vote tally + 1) tally)
+              (Map.empty :: Map (Set Ppm) Int)
```

Comment:
maybe write `(1 + Map.findWithDefault ... ) ` to make the function arguments easier to see?

Reply:
I tried breaking up that big lambda a bit as well.